### PR TITLE
Allow list of commands for uenv activation

### PR DESCRIPTION
**Pull request by teojgo**
https://jira.cscs.ch/browse/VCUE-343

### CODE REVIEW

File: config/systems-uenv/clariden-uenv.py [hunk 1]
```diff
@@ -128,17 +128,28 @@
...
+            raise ConfigError(
+                    f'activation script of {k!r} is not consistent '
+                    f'with the mount point: {image_mount!r}')
+
+        env['prepare_cmds'] = [f'source {activation}']
+    elif isinstance(activation, list):
+        env['prepare_cmds'] = activation
+    else:
+       raise ConfigError(
+           f'activation has to be either a string or list of strings'
```
Comment (bcumming) [inline config/systems-uenv/clariden-uenv.py:145]: Can we change the message to `activation should be a list of commands that configure the environment when run`.
We want to deprecate the single string as an activation script, so let's not mention it in the error message?

**Review by bcumming — CHANGES_REQUESTED** [review]
I have a small requested change to the error message.

A general comment: there is one file per system, each repeating the same code.
It might be time to think of something more generic, so that we don't the scripts don't have to repeat themselves?

**Review by bcumming — APPROVED** [review]

File: config/systems-uenv/clariden-uenv.py (hunk 1 above)
Comment (teojgo) [reply config/systems-uenv/clariden-uenv.py]: I will remove once we have updated the images.

**Review by jgphpc — APPROVED** [review]
tested with:
```
  activation: ['export MPICC=/user-environment/env/default/bin/mpicc']
```

**Pull request merged**